### PR TITLE
chore(eth-multisig-v4): add oas, coredao config

### PR DESCRIPTION
**Pull request by adarsh312**
Ticket: WIN-3695

### CODE REVIEW

File: hardhat.config.ts [hunk 1]
```diff
@@ -134,6 +136,14 @@ const config: HardhatUserConfig = {
       url: `https://bartio.rpc.berachain.com/`,
       accounts: [`${PRIVATE_KEY_FOR_V4_CONTRACT_DEPLOYMENT}`]
     },
+    toas: {
+      url: `https://rpc.mainnet.oasys.games/`,
```
Comment (gianchandania) [inline hardhat.config.ts:140]: This seems to be mainnet rpc url

File: hardhat.config.ts [hunk 2]
```diff
@@ -134,6 +136,14 @@ const config: HardhatUserConfig = {
...
+      accounts: [`${PRIVATE_KEY_FOR_V4_CONTRACT_DEPLOYMENT}`]
+    },
+    tcoredao: {
+      url: `https://rpc.ankr.com/core`,
```
Comment (gianchandania) [inline hardhat.config.ts:144]: This also seems to be mainnet rpc url

File: hardhat.config.ts [hunk 3]
```diff
@@ -227,6 +241,22 @@ const config: HardhatUserConfig = {
           browserURL: 'https://bartio.beratrail.io'
         }
       },
+      {
+        network: 'oasSandVerse',
+        chainId: 20197,
```
Comment (gianchandania) [inline hardhat.config.ts:246]: In this link, the chain id for oasis testnet is 9372. can you confirm which is correct
https://docs.sequence.xyz/solutions/chainlist/non-integrated/oasys-testnet/#rpc-urls

**Review by gianchandania — COMMENTED** [review]
Also please try to use this script in local once to confirm it is working for both networks

File: hardhat.config.ts (hunk 3 above)
Comment (adarsh312) [reply hardhat.config.ts:246]: the reference it taken from the docs https://docs.oasys.games/docs/staking/rpc-endpoint/1-1-rpc-endpoint

File: hardhat.config.ts [hunk 4]
```diff
@@ -227,6 +241,22 @@ const config: HardhatUserConfig = {
...
+        urls: {
+          apiURL: 'https://explorer.testnet.oasys.games/api',
+          browserURL: 'https://explorer.testnet.oasys.games'
+        }
+      },
+      {
+        network: 'coreDao testnet',
```
Comment (gianchandania) [inline hardhat.config.ts:253]: I think here the string should be same as above i.e. coredao

**Review by gianchandania — COMMENTED** [review]
Also make sure the explorer API keys are added to github secrets

**Review by gianchandania — APPROVED** [review]

**Pull request merged**